package.json

```diff
@@ -23,39 +26,77 @@
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "https://github.com/smg-automotive/fe-example-pkg.git"
+    "url": "https://github.com/smg-automotive/components-pkg.git"
+  },
+  "homepage": "https://github.com/smg-automotive/components-pkg#readme",
+  "peerDependencies": {
+    "react": "^17.0.0",
+    "react-dom": "^17.0.0"
   },
-  "homepage": "https://github.com/smg-automotive/fe-example-pkg#readme",
   "devDependencies": {
+    "@babel/core": "7.17.10",
+    "@react-theming/storybook-addon": "1.1.5",
     "@rollup/plugin-commonjs": "22.0.0",
     "@rollup/plugin-node-resolve": "13.3.0",
     "@rollup/plugin-typescript": "8.3.2",
+    "@storybook/addon-actions": "6.5.0-beta.8",
+    "@storybook/addon-controls": "6.5.0-beta.8",
+    "@storybook/addon-docs": "6.5.0-beta.8",
+    "@storybook/addon-essentials": "6.5.0-beta.8",
+    "@storybook/addon-interactions": "6.5.0-beta.8",
+    "@storybook/addon-links": "6.5.0-beta.8",
+    "@storybook/addon-postcss": "2.0.0",
+    "@storybook/builder-webpack5": "6.5.0-beta.8",
+    "@storybook/jest": "0.0.10",
+    "@storybook/manager-webpack5": "6.5.0-beta.8",
+    "@storybook/mdx2-csf": "0.0.3",
+    "@storybook/react": "6.5.0-beta.8",
```

Comment:
Somehow storybook `6.4` doesn't want to work with chakra and custom theming. It also doesn't support react 18 yet so it would block us from updating. `6.5` is in a very good state and will be released soon.